clients/interfaces/command.go

```diff
@@ -0,0 +1,34 @@
+//
```

Comment:
We don't need to duplicate this interface definition here since Go does require the implementations to reference to interface. All the clients of this interface will get the definition from go-mod-core-contracts.